bazel/linux/defs.bzl

```diff
@@ -228,16 +243,24 @@ Available format values are:
             allow_files = True,
             doc = "The list of files that constitute this module. Generally a glob for all .c and .h files. If you use **/* with glob, we recommend excluding the patterns defined by BUILD_LEFTOVERS.",
         ),
+        "cross_compiler": attr.label(
```

Comment:
So, bazel has a native mechanism to provide toolchains for cross compiling - which has several advantages (and of course, some drawbacks). There's a good explanation here: https://bazel.build/extending/toolchains#writing-rules-toolchains

I'm ok with a rule specific way to override a toolchain / compiler, but IMHO the default should come from the standard cpp compiler definition? `cc_common`, https://bazel.build/docs/cc-toolchain-config-reference could be pointers around that.